sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -201,7 +201,16 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
     tempPyFile.delete()
   }
 
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    if (!isTestAtLeastJava9) {
+      prepare()
+    }
+  }
+
   test("backward compatibility") {
+    // TODO Test backward compatibility on JAVA_9 once we have a version supports JAVA_9
```

Comment:
Please file a JIRA and make this IDed TODO, @wangyum .
Otherwise, it's very difficult for the other contributors to pick up this.

Reply:
Done.
[SPARK-28704](https://issues.apache.org/jira/browse/SPARK-28704): Test backward compatibility on JDK9+ once we have a version supports JDK9+